backend/Origam.Gui.Win/Commands/MakeVersionCurrent.cs

```diff
@@ -32,21 +32,25 @@ public class MakeVersionCurrent : AbstractCommand
 	{
 		public override void Run()
 		{
-			WorkbenchSchemaService _schemaService = ServiceManager.Services.GetService(typeof(WorkbenchSchemaService)) as WorkbenchSchemaService;
-			IPersistenceService _persistence = ServiceManager.Services.GetService(typeof(IPersistenceService)) as IPersistenceService;
-			
-			DeploymentVersion version = this.Owner as DeploymentVersion;
-
-			Package ext = _persistence.SchemaProvider.RetrieveInstance(typeof(Package), _schemaService.ActiveExtension.PrimaryKey) as Package;
-
-			Origam.Workbench.Commands.DeployVersion cmd3 = new Origam.Workbench.Commands.DeployVersion();
-			cmd3.Run();
-
-			ext.VersionString = version.VersionString;
-			ext.Persist();
-			_schemaService.ActiveExtension.Refresh();
-			_schemaService.SchemaBrowser.EbrSchemaBrowser.RefreshItem(version.RootProvider);
-			_schemaService.SchemaBrowser.EbrSchemaBrowser.SelectItem(version);
+			var schemaService = ServiceManager.Services.GetService<WorkbenchSchemaService>();
+			DeploymentVersion version = (Owner as DeploymentVersion)!;
+
+			var originalVersion = schemaService.ActiveExtension.VersionString;
+			try
+			{
+				schemaService.ActiveExtension.VersionString = version.VersionString;
+				Origam.Workbench.Commands.DeployVersion cmd3 =
```

Comment:
cmd3 ?

Reply:
It's part of an old code and the refactoring is not aim of the commit.